modules/preview-react/information-highlight/index.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
```

Comment:
I would move this into a file named `InformationHighlight` within the `lib` folder.